frontend-ssr/services/ExerciseSubmissions.service.ts

```diff
@@ -0,0 +1,8 @@
+import HttpService from './Http.service';
+
+export default async function getSubmissions(page: number, query: string) {
+  const responseSubmissions = await HttpService.get(`${process.env.ENDPOINT}/exercises/submissions?search=al`)
```

Comment:
We should use the params when creating the GET URL. Also I think we should have default values for page as well as query (0 and `''`)